tests/conftest.py

```diff
@@ -172,6 +172,7 @@ def pytest_configure(config):
     # pylint: disable=unused-variable
     if config.webengine:
         import PyQt5.QtWebEngineWidgets
+    # pylint: enable=unused-variable
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.